monix-execution/shared/src/main/scala/monix/execution/schedulers/TestScheduler.scala

```diff
@@ -15,53 +15,151 @@
  * limitations under the License.
  */
 
-package monix.execution.schedulers
+package monix.execution
+package schedulers
 
-import monix.execution.Cancelable
 import monix.execution.atomic.AtomicAny
 import monix.execution.cancelables.SingleAssignCancelable
 import monix.execution.misc.NonFatal
 import monix.execution.schedulers.TestScheduler._
-// Prevents conflict with the deprecated symbol
-import monix.execution.{ExecutionModel => ExecModel}
 
 import scala.annotation.tailrec
 import scala.collection.immutable.SortedSet
 import scala.concurrent.duration.{Duration, FiniteDuration, TimeUnit}
 import scala.util.Random
 
-/** A scheduler meant for testing purposes. */
+/** [[Scheduler]] implementation meant and a provider of
```

Comment:
I think it could be worded a bit better, maybe without "implementation meant" phrase?